target/linux/qualcommax/patches-6.6/100-arm64-dts-qcom-ipq6018-add-pwm-node.patch

```diff
@@ -0,0 +1,25 @@
+--- a/arch/arm64/boot/dts/qcom/ipq6018.dtsi
++++ b/arch/arm64/boot/dts/qcom/ipq6018.dtsi
+@@ -435,8 +435,21 @@
+ 		};
+ 
+ 		tcsr: syscon@1937000 {
+-			compatible = "qcom,tcsr-ipq6018", "syscon";
++			compatible = "qcom,tcsr-ipq6018", "syscon", "simple-mfd";
+ 			reg = <0x0 0x01937000 0x0 0x21000>;
++			ranges = <0x0 0x0 0x01937000 0x21000>;
```

Comment:
Hm, this looks incorrect to me

Reply:
Though it seems to be correct:
https://patches.linaro.org/project/linux-devicetree/patch/20231005043127.2690639-5-quic_devipriy@quicinc.com/

Use the latest patch please, and keep it whole with the full header so git am can be used to apply it and we actually know what its about